clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -2331,6 +2335,9 @@ def err_init_reference_member_uninitialized : Error<
   "reference member of type %0 uninitialized">;
 def note_uninit_reference_member : Note<
   "uninitialized reference member is here">;
+def warn_field_requires_explicit_init : Warning<
+  "field %select{%1| in %1}0 is not explicitly initialized, but was marked as "
```

Comment:
```suggestion
  "field %select{|in }0 %1 is not explicitly initialized, but was marked as "
```

Reply:
Also, looks like a double-space between 'field' and 'in'?  Shoudl probably make sure that is right.  Also, do you have tests for this?  I don't see this syntax in any of them via browser search.